recipes/autogenes/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "1.0.4" %}
+
+package:
+  name: autogenes
+  version: {{ version }}
+
+source:
+  url:
+    - https://files.pythonhosted.org/packages/f0/04/4768c40962c38dd15b6cbd67d23f1c0d97d2dedeaa6558f6f8689458b4c5/autogenes-{{ version }}.tar.gz
```

Comment:
I think this URL won't work once the version gets bumped... 
It usually looks more like this `https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz`